src/solvers/refinement/string_refinement.cpp

```diff
@@ -1882,12 +1880,11 @@ static exprt instantiate(
 /// entailed by a (transformed) axiom.
 /// \param [in] axiom: the axiom to instantiate
 /// \param index_set: set of indexes
-/// \param generator: constraint generator object
+/// \param witnesses: ???
```

Comment:
@romainbrenguier @allredj provide a description for `witnesses`?

Reply:
```
`axiom`'s witnesses for non containment
```

Reply:
Thanks, added!